src/main/java/mod/linguardium/experimentaway/mixin/MinecraftClientMixin.java

```diff
@@ -1,25 +1,22 @@
 package mod.linguardium.experimentaway.mixin;
 
-import com.mojang.serialization.Lifecycle;
 import net.fabricmc.api.EnvType;
 import net.fabricmc.api.Environment;
 import net.minecraft.client.MinecraftClient;
-import net.minecraft.world.SaveProperties;
-import net.minecraft.world.level.storage.LevelStorage;
-import org.objectweb.asm.Opcodes;
-import org.spongepowered.asm.mixin.Final;
 import org.spongepowered.asm.mixin.Mixin;
-import org.spongepowered.asm.mixin.Shadow;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.ModifyVariable;
-import org.spongepowered.asm.mixin.injection.Redirect;
 
 @Environment(EnvType.CLIENT)
 @Mixin(MinecraftClient.class)
 public abstract class MinecraftClientMixin {
 
-    @ModifyVariable(at=@At(value="FIELD", target="Lnet/minecraft/client/MinecraftClient$WorldLoadAction;NONE:Lnet/minecraft/client/MinecraftClient$WorldLoadAction;", opcode = Opcodes.GETSTATIC, ordinal=0), method="startIntegratedServer(Ljava/lang/String;Lnet/minecraft/util/registry/DynamicRegistryManager$Impl;Ljava/util/function/Function;Lcom/mojang/datafixers/util/Function4;ZLnet/minecraft/client/MinecraftClient$WorldLoadAction;)V",name="bl2")
-    private boolean pretendWeAreStable(boolean bl2) {
+    @ModifyVariable(at = @At(value = "STORE"),
```

Comment:
I changed the formatting here to make it more readable. Hopefully you don't mind.